### PR TITLE
Broadband speed NUTS 2 & 3 

### PR DESCRIPTION
**Pull request by dede95**
Branch for issue #103 :

- Data for indicators
- Schemas for datasets
- Notebook for broadband indicator

### CODE REVIEW

File: ds/data/processed/broadband/broadband_download_speed_data.nuts2.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    description: Broadband download speed.
+    format: .3f
+    id: broadband_download_speed_data
+    label: Download Speed
+    data_type: float
```
Comment (mindrones) [inline ds/data/processed/broadband/broadband_download_speed_data.nuts2.yaml]: I'd add a new type in `schema/types.yaml` like, say:

```yaml
BitTransferRate:
  kind: bit transfer rate
  data_type: float
  unit_string: Mb/s
  type: Unit
```

and here I'd remove `data_type: float`, using `type: BitTransferRate` instead.

File: ds/data/processed/broadband/broadband_download_speed_data.nuts3.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  nuts_id:
+    type: NutsRegion.nuts_id
+  nuts_year_spec:
+    type: NutsRegion.nuts_year_spec
+  value:
+    description: Broadband download speed.
+    format: .3f
+    id: broadband_download_speed_data
+    label: Download Speed
+    data_type: float
```
Comment (mindrones) [inline ds/data/processed/broadband/broadband_download_speed_data.nuts3.yaml]: here too I'd use `type: BitTransferRate`

**Review by mindrones — CHANGES_REQUESTED** [review]
Some changes required. If it's not a lot of work I'd move this code to scripts so to avoid the need for future refactors :)

**Comment by mindrones** [comment]
@dede95 Alternatively, you could just add `unit_string: Mb/s` (I added it recently, see https://github.com/nestauk/beis-indicators/blob/dev/ds/data/schema/schema_template.yaml#L41)

**Comment by dede95** [comment]
@mindrones Added the `BitTransferRate` type to schema/types.yaml and replaced `data_type` with `type`

**Review by mindrones — DISMISSED** [review]
Nice, good to merge.

**Review by mindrones — APPROVED** [review]
No, sorry I've changed my mind, I'll run my scripts after #227 will be in too, as I'll need to filter out `.lep.csv` files.
Good to merge.

**Pull request merged**